custom_components/tuya_local/devices/klarstein_wonderwall_air.yaml

```diff
@@ -0,0 +1,118 @@
+name: Klarstein Wonderwall Air
+products:
+  - id: cl2wgyhohpjjjuzu
+    name: Wonderwall Art Smart
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 2
+      name: temperature
+      type: integer
+      mapping:
+        - scale: 10
+          step: 10
+      range:
+        min: 0
+        max: 370
+      unit: "\u00b0C"
+    - id: 3
+      name: current_temperature
+      type: integer
+      mapping:
+        - scale: 10
+          step: 10
+      range:
+        min: 0
+        max: 500
+      unit: "\u00b0C"
+    - id: 4
+      type: string
+      mapping:
```

Comment:
Presets should generally be taken from existing translations.  In this case I think the best match is to add `translation_key: thermostat` between `entity` and `dps` lines, and use `manual` and `program` from https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/translations/en.json#L94 and `away` from https://github.com/home-assistant/core/blob/dev/homeassistant/components/climate/strings.json#L88

Reply:
Alternatively, manual could be mapped to `none` from the default presets, and auto to `home` or `activity`.

hvac_mode could also be adjusted to report `auto` when preset is auto, though that complicates the config and some devices do not like receiving both on and preset at the same time (which is what would result)

Reply:
First option done.